README.md

```diff
@@ -240,6 +240,16 @@ spec:
       targetValue: 10 # this will be treated as targetAverageValue
 ```
 
+### Metric weighting based on backend
+
+Skipper supports sending traffic to different backend based on annotations present on the
+`Ingress` object. When the metric name is specified without a backend as `requests-per-second`
+then the number of replicas will be calculated based on the full traffic served by that ingress.
+If however only the traffic being routed to a specific backend should be used then the
+backend name can be specified as a metric name like `requests-per-second|backend1` which would
```

Comment:
Is this correct? `requests-per-second|backend1` in the code it's splitting at `,` afaict. I dind't know you could even use characters like this I thought it was limited to alphanumeric plus `-` or something like that.

Reply:
Yep, actually I left it like this to ask you which separator we should use. I ran my tests using the comma separator and it worked. It's just the name of the metric and I don't think it follows any rules like k8s resource rule names.

Reply:
I like `,` separators more than `|`.

Reply:
What about `/` if it's an option?

Reply:
Slashed are also possible I guess.